Formula/bgfx.rb

```diff
@@ -0,0 +1,60 @@
+class Bgfx < Formula
+  desc "Cross-platform, graphics API agnostic, rendering library"
+  homepage "https://github.com/bkaradzic/bgfx"
+  url "https://github.com/bkaradzic/bgfx.cmake.git", tag: "v1.115.7904-f5ec641", revision: "f5ec6410770d620cca99a94a5ab26c7bfb63bb68"
+  version "1.115.7904"
+  license any_of: ["CC0-1.0", "BSD-2-Clause"]
+  head "https://github.com/bkaradzic/bgfx.cmake.git", branch: "master"
+
+  depends_on "cmake" => :build
+
+  on_linux do
+    depends_on "gcc"
```

Comment:
Actually, we might not need this anymore, since it did successfully use GCC 5 to build on Linux.